frontend/packages/console-app/src/components/access-modes/access-mode.tsx

```diff
@@ -69,24 +68,19 @@ export const AccessModeSelector: React.FC<AccessModeSelectorProps> = (props) =>
       {loaded &&
         allowedAccessModes &&
         accessModeRadios.map((radio) => {
-          let radioObj = null;
           const disabled = !allowedAccessModes.includes(radio.value);
-          allowedAccessModes.forEach((mode) => {
-            const checked =
-              !disabled && !loadError ? radio.value === accessMode : radio.value === mode;
-            radioObj = (
-              <RadioInput
-                {...radio}
-                key={radio.value}
-                onChange={onAccessModeChange}
-                inline
-                disabled={disabled}
-                checked={checked}
-                name="accessMode"
-              />
-            );
-          });
-          return radioObj;
+          const checked = radio.value === accessMode;
+          return (
+            <RadioInput
+              {...radio}
+              key={radio.value}
+              onChange={onAccessModeChange}
+              inline
+              disabled={disabled}
+              checked={checked}
+              name="accessMode"
+            />
```

Comment:
`accessMode` is `""` then what will be checked?

Reply:
the 'ReadWriteOnce' will be checked..

Reply:
sounds good!